platforms/gke/base/core/gke_enterprise/configmanagement/git/feature.tf

```diff
@@ -17,7 +17,7 @@ resource "google_gke_hub_feature" "configmanagement" {
 
   location = "global"
   name     = "configmanagement"
-  project  = google_project_service.mesh_googleapis_com.project
+  project  = google_project_service.anthosconfigmanagement_googleapis_com.project
 
   fleet_default_member_config {
     configmanagement {
```

Comment:
Shall we keep this `configmanagement` in sync with  the one below to avoid surprising differences in configuration?